cmd/commands/cli/command.go

```diff
@@ -291,6 +291,18 @@ func (c *cliApp) connect(args []string) (err error) {
 	}
 
 	consumerID, providerID, serviceType := args[0], args[1], args[2]
+	migrationStatus, err := c.tequilapi.MigrateHermesStatus(consumerID)
+	if migrationStatus.Status == contract.MigrationStatusRequired {
```

Comment:
I think maybe we should just print an error?

Reply:
I thought about it. But what about programatically run nodes? @tomasmik

Reply:
This is a double edged sword, both ways have their drawbacks, just printing `Migration required` and not doing it might be inconvenient for a consumer and doing the migration for the can result in some funky business by accident. So I'll let you chose.

Reply:
To add: this is required migration, but not optional